Lib/test/test_plistlib.py

```diff
@@ -852,6 +853,29 @@ def test_dump_aware_datetime(self):
         s = plistlib.dumps(dt, aware_datetime=True)
         self.assertIn(b"1234-05-06T15:01:07Z", s)
 
+    def test_dump_utc_aware_datetime(self):
+        dt = datetime.datetime(2345, 6, 7, 8, tzinfo=datetime.UTC)
+        s = plistlib.dumps(dt, fmt=plistlib.FMT_XML, aware_datetime=True)
+        self.assertIn(b"2345-06-07T08:00:00Z", s)
+
+    @unittest.skipUnless("America/Los_Angeles" in zoneinfo.available_timezones(),
+                         "Can't find timezone datebase")
+    def test_dump_aware_datetime_without_aware_datetime_option(self):
```

Comment:
Was an aware datetime with UTC timezone tested? If not, add a test. It is better to have something if timezone datebase does not work.

Reply:
That's ` test_dump_utc_aware_datetime_without_aware_datetime_option` just below this test.

Reply:
Yes, this is added after Serhiy's review, I forgot to resolve it.